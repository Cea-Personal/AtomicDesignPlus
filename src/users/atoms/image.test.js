import React from "react";
import {render,cleanup} from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import Image from "./image";

afterEach(cleanup)
it('displays an image',()=>{
    const {getByRole, asFragment, container}= render(
        <Image/>
    )
    const ImageContent =getByRole('img')
    expect(container).toBeDefined()
    expect(ImageContent).toBeDefined();
    expect(asFragment()).toMatchSnapshot()
})