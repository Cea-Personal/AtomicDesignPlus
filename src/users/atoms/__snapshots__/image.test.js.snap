// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`displays an image 1`] = `
<DocumentFragment>
  <div
    class="sc-bdVaJa hKwztZ"
  >
    <img
      class="sc-bwzfXH Cjhqx"
      src="landing-page-girl.png"
    />
  </div>
</DocumentFragment>
`;
